src/content/guides/how-to-create-an-ethereum-account/index.md

```diff
@@ -49,20 +47,21 @@ Once you have saved your recovery phrase you should see your wallet dashboard wi
 
 ### Are my wallet and my Ethereum account the same?
 
-No, just like online banking you can have many different accounts all stored in one wallet application. Your 12 or 24 word phrase secures them all: it is like the seed of a big tree (which is why you must keep it safe). Each branch of the tree holds a key, and every key is one of your accounts. If you lose access to the wallet (i.e. your tree gets cut down), you can always restore all your different accounts using other software and the same seed (which will always grow the same tree).
+No. Each account has a unique Ethereum address and a secret key. The secret key is used to sign and send transactions. The wallet is a management tool that helps you to manage accounts. A single wallet might give access to several accounts, and a single account can be accessed by multiple wallets.
 
 ### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
 
 No, you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping models and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
 
 ### If I own an ETH address, do I own the same address on other blockchains?
 
-You can use the same address on all EVM compatible blockchains (if you have the type of wallet with a recovery phrase). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains, like Bitcoin, implement a completely separate set of network rules and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on which blockchains are supported.
+You can use the same address on all EVM compatible blockchains. This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains, like Bitcoin, implement a completely separate set of network rules and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on which blockchains are supported.
```

Comment:
Hate to do it... but what is the EVM??

🤔 I'm not sure the best way to explain this with clarity and brevity... I don't think just spelling out "Ethereum Virtual Machine" is going to help much here, but we should definitely assume people on this page do not know what the EVM is. 

Maybe something like:

> Some blockchains are built using similar underlying software to Ethereum, known as the Ethereum Virtual Machine, which enables account compatibility with these chains.

Reply:
^ I chose the word "similar" here instead of "same" since that could falsly equivocate these networks to Ethereum.

Reply:
```suggestion
You can use the same address on all blockchains that use similar underlying software to Ethereum (known as 'EVM-compatible'). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains, like Bitcoin, implement a completely separate set of network rules and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on which blockchains are supported.
```